src/api/devices.ts

```diff
@@ -14,39 +14,7 @@ const sortDevices = (devices: Device[]) =>
     }
   })
 
-export const SHARED_DEVICE = 'Shared Device'
```

Comment:
We can come back and re-add if we decide we really want to show in the sidebar when viewing a shared device, but we're already about to remove the route listing, so probably not a big deal to match old connect

Reply:
ACK